### DISCUSSION TITLE
"Accept All Cookies" or "Customize" covering panel

### DISCUSSION THREAD

**Discussion by obit11** [Cookie-AutoDelete/Cookie-AutoDelete#1197] [Q&A / Support]
I don't know if the behavior I'm describing is because I use this extension. Several sites put up a box when I go to their page, it covers a good part of the page, and offers me either a quick "Accept All Cookies" or go into a customize mode where it shows "strictly necessary" selected and greyed out, and several others where you can enable or disable. I pick disable on those.

That gets rid of the covering box this time, but I keep getting it again with later visits to this or related pages. Stack Exchange is one of these site, but there are others.

I tried whitelisting, hope that it would keep a cookie that remembered my choice but that seems to do nothing.I

Again, I don't know if this is because I'm using Cookie-AutoDelete. At any rate I'm looking for a solution.

**Comment by kennethtran93** [comment]
Whitelisting should normally take care of keeping any cookies from that site, though it would depend on how it was configured.  Removal of those panels usually requires a cookie to be set on that site (and stays set).

Are you using Firefox?  If so:
- Are you using containers (including temporary container webextensions)?
- Any other cookie or other site data removing webextensions?

**Reply by obit11** [reply]
I am not using containers. Nothing else specifically about cookies. I am running ublock origin, bypass paywalls, and  ghostery.

**Reply by kennethtran93** [reply]
A quick glance at bypass paywall does remove cookies to some extent, so depending on its configuration it may be removed from their extension.

On Firefox's internal site cookie manager (Privacy & Security > Cookies and Site Data > Manage Exceptions), did you explicitly set up sites to block/allow?  Is the 'Clear Cookies and Site Data when Firefox is closed' unchecked? What is your 'Enhanced Tracking Protection' setting at?

I'm also not entirely sure how sites store the cookie preferences, as the available data options are cookie, localstorage, indexedDB, etc.  What cleaning options have you enabled in CAD, and what is the whitelist expression you used and its options?

**Reply by obit11** [reply]
I'm using google chrome (on Linux), not firefox.

**Reply by kennethtran93** [reply]
Apologies for the delay in responding.

- What sort of whitelist configuration did you do and what is your browser setting (more notably the settings in `chrome://settings/cookies`) and CAD settings at?
- Are you having Chrome remove all cookies and site data every time you close all windows / restart browser?
- Are you using Chrome's incognito mode / private browsing as your everyday usage?
